src/views/Home.jsx

```diff
@@ -1,11 +1,29 @@
 import './Home.css';
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import { useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
+
+export function Home({ listToken, setListToken }) {
+	const navigate = useNavigate();
+
+	const createToken = async () => {
+		const token = await generateToken();
+
+		setListToken(token);
+	};
```

Comment:
The button is working properly, since once I clicked on it, I was redirected to the list. Just making sure... did you confirm that it sets a new listToken?
I think we won't be able to see if it's fully functional until we start sending new data to the database (After we merge both our files)

Reply:
The new token will be stored to listToken when you click on the button, and the useEffect hook we have in the Home view will be listening for changes to the listToken (from null to there being a token) before it triggers navigate and redirects the user to the list view. I've tested this by logging the token to the console as well as checking localStorage.

Reply:
Awesome sauce! I wasn't sure how to see it. I'm using Chrome and Jeff told me Firefox is the one that can show the localStorage. So we're good here :D

Reply:
For Chrome you can go to Inspect > Application tab up top > Local Storage tab to your left